crazyflie-link/src/context.rs

```diff
@@ -69,14 +70,24 @@ impl LinkContext {
             Err(_) => return Err(Error::InvalidUri),
         };
 
-        let mut safelink = true;
+        let mut flags = ConnectionFlags::SAFELINK | ConnectionFlags::ACKFILTER;
         for (key, value) in uri.query_pairs() {
-            if key == Cow::Borrowed("safelink") {
-                safelink = value == Cow::Borrowed("1");
-            }
+            match key {
+                Cow::Borrowed("safelink") => {
```

Comment:
Using the Cow object in not needed if we match "key.as_ref()" and the comparison can be done to a &str directly (since cow is a smart pointer, it implements Deref).